plugins/regex_remap/regex_remap.cc

```diff
@@ -461,8 +475,9 @@ RemapRegex::compile(const char **error, int *erroffset)
 
       if (ix > -1) {
         if ((ix < 10) && (ix > ccount)) {
-          TSDebug(PLUGIN_NAME, "Trying to use unavailable substitution, check the regex!");
-          return -1; // No substitutions available other than $0
+          error = "using unavailable captured substring ($n) in substitution";
+          TSDebug(PLUGIN_NAME, error);
```

Comment:
I know this was in the original code, but is logging this here useful? I would think it would be better to let it go the caller for better description.